lib/GridItem.jsx

```diff
@@ -396,16 +394,15 @@ export default class GridItem extends React.Component<Props, State> {
       <Resizable
         draggableOpts={{
           disabled: !isResizable,
-          nodeRef: this.elementRef
```

Comment:
@STRML  It seems that this code is not correct, here we pass the reference of the GridItem that will be used in DraggableCore as being a reference of a handle